draft-ietf-httpbis-incremental.md

```diff
@@ -141,10 +138,32 @@ intermediaries SHOULD respond with a 503 Service Unavailable error, accompanied
 by a connection_limit_reached Proxy-Status response header field
 ({{Section 2.3.12 of PROXY-STATUS}}).
 
+For performance and efficiency reasons, a small amount of buffering might be used
```

Comment:
Could we add some bounds to the "small amount", so that clients could have an expectations of what's allowed?

While I don't expect this to be an issue for Chunked OHTTP, other legitimate users of this feature (e.g. gRPC) might be sending really small messages that could be buffered at the intermediaries even with `Incremental: always` if there is no consensus around what's considered a "small amount".

Reply:
That's a good point.

The text right below says intermediaries might "set limits on the number bytes that are buffered or the time that buffers are held before forwarding."

I think the latter (delaying delivery for a short period of time) is fine. But it might not be a good idea to allow intermediaries buffer some bytes indefinitely.

Reply:
The goal here is to forward when *either* condition is met, not both.  That's worth saying.

Reply:
Updated with something like that.

Reply:
Thank you!